modules/cimg/CMakeLists.txt

```diff
@@ -103,4 +77,44 @@ ivw_register_license_file(NAME tiff TARGET inviwo::tiff MODULE CImg
     FILES ${IVW_EXTENSIONS_DIR}/tiff/COPYRIGHT
 )
 
+
+
+option(IVW_CIMG_USE_OPENEXR  "Add OpenEXR image format support (HDR images) " ON)
```

Comment:
Should probably state that there is a problem with exr and that is why it is optional

Reply:
So you want to add a message explaining why one has to exclude OpenEXR when packaging Inviwo?

Reply:
Something inline with that. At least state what type of a problem there is and that is why we need this workaround.